### PR TITLE
Remove unused data files

### LINKED ISSUE #9: Remove preprocessing-specific data files
I don't know which files are only used for preprocessing, but I would like to delete anything we don't need.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #9.

## Changes proposed in this pull request

- Remove any data files that appear to be unused by QSIRecon.

### CODE REVIEW

File: qsirecon/data/FreeSurfer2dseg.txt
Comment (mattcieslak) [inline qsirecon/data/FreeSurfer2dseg.txt:1]: This one is used in the 5tt hsvs

File: qsirecon/data/FreeSurferColorLUT.txt
Comment (mattcieslak) [inline qsirecon/data/FreeSurferColorLUT.txt:1]: I think 5tt hsvs also needs this one

File: qsirecon/data/affine.json
Comment (mattcieslak) [inline qsirecon/data/affine.json:1]: For now we won't need this file, but someday we may want to register data to the preprocessed dmri and summarize it within the bundles (like hbcd qsm data). We should make an issue for this

Comment (mattcieslak) [reply qsirecon/data/affine.json:1]: you know what, this will still be present in qsiprep so we can delete it here

File: tests/data_tests.py
Comment (mattcieslak) [inline tests/data_tests.py:1]: this whole file can go

**Review by mattcieslak — CHANGES_REQUESTED** [review]

File: qsirecon/data/FreeSurfer2dseg.txt
Comment (tsalo) [reply qsirecon/data/FreeSurfer2dseg.txt:1]: Added back

File: qsirecon/data/FreeSurferColorLUT.txt
Comment (tsalo) [reply qsirecon/data/FreeSurferColorLUT.txt:1]: Added back

**Review by mattcieslak — APPROVED** [review]

**Pull request merged**